### PR TITLE
Arbitrum liquidation bot scenario updates

### PR DESCRIPTION
**Pull request by scott-silver**

### CODE REVIEW

**Comment by scott-silver** [comment]
rebased to incorporate the changes from #748

File: deployments/arbitrum/usdc/deploy.ts [hunk 1]
```diff
@@ -16,6 +16,7 @@ export default async function deploy(deploymentManager: DeploymentManager, deplo
   const GMX = await deploymentManager.existing('GMX', '0xfc5a1a6eb076a2c7ad06ed22c90d7e710e35ad0a', 'arbitrum');
   const WETH = await deploymentManager.existing('WETH', '0x82af49447d8a07e3bd95bd0d56f35241523fbab1', 'arbitrum');
   const WBTC = await deploymentManager.existing('WBTC', '0x2f2a2543b76a4166549f7aab2e75bef0aefc5b0f', 'arbitrum');
+  const ARB = await deploymentManager.existing('ARB', '0x912ce59144191c1204e64559fe8253a0e49e6548', 'arbitrum');
```
[the remark this answers is not in the document]
Comment (kevincheng96) [reply deployments/arbitrum/usdc/deploy.ts:19]: Oh ARB is already declared on line 15.

[the remark this answers is not in the document]
Comment (scott-silver) [reply deployments/arbitrum/usdc/deploy.ts]: deleted

**Comment by kevincheng96** [comment]
Hmm, seeing some `TF` errors in the Arbitrum liquidation scenarios. Might need to either switch a pool or adjust some of the numbers.

**Comment by scott-silver** [comment]
Made some updates in the Arbitrum scenarios. The amount of USDC that we're able to borrow in a flash loan seems to have dropped, so we were hitting TF (Transfer Failure) errors from Uniswap because we were trying to borrow more than the pool could lend.

File: scenario/LiquidationBotScenario.ts [hunk 2]
```diff
@@ -621,103 +669,129 @@ scenario(
...
+  };
+
+  scenario(
+    `LiquidationBot > reverts when price slippage is too high`,
+    {
+      upgrade: {
+        targetReserves: exp(20_000, 18)
       },
-      betty: { $base: 1000 },
+      filter: async (ctx) => matchesDeployment(ctx, [{network: 'mainnet'}]),
```
Comment (kevincheng96) [inline scenario/LiquidationBotScenario.ts:698]: Was it intentional to remove `arbitrum` from this list? If so, we could remove the `arbitrum` key in the `baseTokenBalances` and `assetAmounts` mapping.

**Review by kevincheng96 — APPROVED** [review]
Looks good!

File: scenario/LiquidationBotScenario.ts (hunk 2 above)
Comment (scott-silver) [reply scenario/LiquidationBotScenario.ts:692]: yeah, it's intentional. we can not borrow a large enough sum from the flash loan pool to create enough slippage to make this test revert.

deleted the `arbitrum` keys from `baseTokenBalances` and `assetAmounts`

**Pull request merged**